### PR TITLE
MAgPIE coupling: adapt start scripts to MAgPIE

### PR DESCRIPTION
**Pull request by dklein-pik**
## Purpose of this PR

Rename switch according to changes in MAgPIE and change year until which GHG prices are usually muted from 2020 to 2030 (as it is default now in MAgPIE).

Bugfix: let subsequent runs wait also if a titletag is provided. Therefore, use iscenarios (where the tag has already been appended to the scenario name) instead of isettings (which contains the original scenario names without the tag) in configureCfg.R.

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Bug fix 
- [ ] Refactoring
- [ ] New feature 
- [ ] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [ ] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ ] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [ ] All automated model tests pass (`FAIL 0` in the output of `make test`)

## Further information (optional):

* Test runs are here: 
* Comparison of results (what changes by this PR?):

### CODE REVIEW

**Review by LaviniaBaumstark — APPROVED** [review]

**Comment by orichters** [comment]
Your changes look good, thanks for fixing that. On the other hand, I don't understand why `isettings` was used here in the first place. Why was it implemented the way it was if `iscenarios` and `isettings` are basically redundant. Do you have an idea, @dklein-pik?

File: scripts/start/configureCfg.R [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
 
-configureCfg <- function(icfg, iscen, iscenarios, isettings, verboseGamsCompile = TRUE) {
```
[the remark this answers is not in the document]
Comment (orichters) [reply scripts/start/configureCfg.R:8]: according to reposearch, that should be fine.

**Comment by LaviniaBaumstark** [comment]
thy are not identical, iscenarios is the subset of runs which should be started now while isettings still contains all infos from the scenario_config. But no idea, why this was used here

**Review by orichters — APPROVED** [review]
I checked [start.R before I touched anything in it](https://github.com/remindmodel/remind/blob/ee88765f55af612bf2529fa5d7c4c4766e6f0b96/start.R), and the current version of configureCfg. I see no reason this change could break anything, as only `isettings[iscen, …]` is the only row ever used, as far as I see. And you, David, [seem to have implemented](https://github.com/remindmodel/remind/commit/6c3ce254185cda87e423d7ab0aa7bf591f95e8c6) it as such, so you should know if there was a specific reason to do it like that ;)

**Comment by orichters** [comment]
> thy are not identical, iscenarios is the subset of runs which should be started now while isettings still contains all infos from the scenario_config. But no idea, why this was used here

Yes, that is true, but if you only call the `iscen` row, I don't see a difference.

File: scripts/start/configureCfg.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (dklein-pik) [reply scripts/start/configureCfg.R:8]: I also didn't find any further occurrences.

**Comment by dklein-pik** [comment]
> I checked [start.R before I touched anything in it](https://github.com/remindmodel/remind/blob/ee88765f55af612bf2529fa5d7c4c4766e6f0b96/start.R), and the current version of configureCfg. I see no reason this change could break anything, as only `isettings[iscen, …]` is the only row ever used, as far as I see. And you, David, [seem to have implemented](https://github.com/remindmodel/remind/commit/6c3ce254185cda87e423d7ab0aa7bf591f95e8c6) it as such, so you should know if there was a specific reason to do it like that ;)

Thanks for linking the old code. I looked into it but cannot remember why we took `isettings`, since already back then we only used `isettings[iscen,...]` which is the same as `iscenarios[iscen,...]` as you said.

**Pull request merged**